src/pages/transactions/add-transaction.tsx

```diff
@@ -0,0 +1,72 @@
+import { Button, Dialog, DialogPanel, DialogTitle } from "@headlessui/react";
+import { useState } from "react";
+import { AiOutlineArrowLeft } from "react-icons/ai";
+import { TransactionItem } from "./transaction-item";
+
+const categories: string[] = [
+  "Salary",
+  "Food",
+  "Transport",
+  "Entertainment",
+  "Sellings",
+  "Tech",
+];
+
+export default function AddTransactionModal() {
+  let [isOpen, setIsOpen] = useState(false);
+
+  function open() {
+    setIsOpen(true);
+  }
+
+  function close() {
+    setIsOpen(false);
+  }
+
+  return (
+    <>
+      <Button
+        onClick={open}
+        className="bg-green text-beige font-bold w-40 rounded-full py-2 px-4 text-base focus:outline-none data-[focus]:outline-1 data-[focus]:outline-white"
+      >
+        Add Transaction
+      </Button>
+
+      <Dialog
+        open={isOpen}
+        as="div"
+        className="relative z-10 focus:outline-none"
+        onClose={close}
+      >
+        <div className="fixed inset-0 z-10 w-screen overflow-y-auto">
+          <div className="flex min-h-full items-center justify-center text-navy">
+            <DialogPanel
+              transition
+              className="w-full h-screen rounded-xl bg-beige p-4 backdrop-blur-2xl duration-300 ease-out data-[closed]:transform-[scale(95%)] data-[closed]:opacity-0"
+            >
+              <AiOutlineArrowLeft className="text-3xl my-2" onClick={close} />
+
+              <DialogTitle>
```

Comment:
It seems that this element is a h2 internally, avoid using h1 inside a DialogTitle
![Captura de pantalla 2024-08-12 154926](https://github.com/user-attachments/assets/f1d7571c-6e48-4071-bbdb-2c8d59202bc9)